src/features/metadata-based-view/MetadataBasedItemList.scss

```diff
@@ -89,6 +89,7 @@ $bdl-MetadataBasedItemList-insetShadowRight: 3px 0 3px -2px $bdl-gray-10 inset;
     color: $bdl-gray;
 
     & .btn-plain {
+        text-align: left;
```

Comment:
It would be good to create a Storybook story + visual regression test for this feature in a later PR, so that we can verify CSS changes